tests/integration/heartbeat.rs

```diff
@@ -0,0 +1,20 @@
+use crate::utils::{send_get_request, spawn_app};
+
+#[tokio::test]
+async fn index_get() {
+    let app = spawn_app().await;
+    let r = send_get_request(&app, "/").await;
+    assert_eq!(r.status(), 200);
+    let body = r.text().await.expect("Response body missing.");
+    assert_eq!(body, "Hello world!");
+}
+
+#[tokio::test]
+async fn heartbeats_get() {
+    let app = spawn_app().await;
+    let test_cases = vec!["/__heartbeat__", "/__lbheartbeat__"];
```

Comment:
Some question here about the use of `vec!` as elsewhere.

Reply:
yes

Reply:
c65191e

Reply:
c65191e